server/src/main/java/org/elasticsearch/index/shard/InternalIndexingStats.java

```diff
@@ -145,7 +145,8 @@ IndexingStats.Stats stats(
                 noopUpdates.count(),
                 isThrottled,
                 TimeUnit.MILLISECONDS.toMillis(currentThrottleMillis),
-                timeSinceShardStartedInNanos > 0 ? (double) totalIndexingTimeSinceShardStarted / timeSinceShardStartedInNanos : 0
+                timeSinceShardStartedInNanos > 0 ? (double) totalIndexingTimeSinceShardStarted / timeSinceShardStartedInNanos : 0,
```

Comment:
I might be catching up slowly - but could we not keep this as `totalIndexingTimeSinceShardStarted` here? Since then when adding stats together we would be combining them fairly.

For instance if one copy has a load of 10 for a duration of 1, whereas another has a load of 1 for a duration of 10. Then the combined result will be a load of 11 for a duration of 11. But in reality, the load was just a little shy of 2 on avg.

I realize this concerns the previous PR, but it becomes somewhat more obvious now.

I think we do not rely on the result of add currently so I'd be ok to postpone changing this.

Reply:
That's a good point, we can defer the computation until needed. I'll change it, thanks for catching it!